modules/claims_api/lib/claims_api/form_schemas.rb

```diff
@@ -4,8 +4,13 @@
 
 module ClaimsApi
   class FormSchemas < JsonSchema::FormSchemas
+    def initialize(schema_version: 'v1')
+      # super
```

Comment:
@FonzMP Rubocop wants 'super' here, but that breaks SO many things

Reply:
interesting. How is the Appeals API version bypassing this?